spec/system/visit_users_details_page.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'Update user Page', type: :system do
+  include CognitoSupport
+
+  let(:member_user_id) { SecureRandom::uuid }
+  let(:member_first_name) { 'Tester' }
+  let(:member_family_name) { 'Testerator' }
+  let(:member_email) { 'test@test.com' }
+
+  let(:cognito_user) {
+    { username: member_user_id,
+      user_attributes: [{name: "given_name", value: member_first_name},
+                        {name: "family_name", value: member_family_name},
+                        {name: "email", value: member_email},
+                        {name: "custom:roles", value: "certmgr"}]}
+  }
+
+  let(:cognito_users) {
+    { users: [
+      { username: member_user_id,
+        attributes: [{name: "given_name", value: member_first_name},
+                          {name: "family_name", value: member_family_name},
+                          {name: "email", value: member_email},
+                          {name: "custom:roles", value: "certmgr"}]}
+      ]}
+    }
+
+  before(:each) do
+    stub_cognito_response(method: :admin_get_user, payload: cognito_user)
+  end
+
+  context 'GDS user' do
+    before(:each) do
+      login_gds_user
+    end
+
+    it 'renders the page' do
+      visit update_user_path(user_id: member_user_id)
+      expect(page).to have_title t('users.show.title', name: member_first_name + ' ' + member_family_name)
+      expect(page).to have_content member_first_name + ' ' + member_family_name
+    end
+  end
+
+  context 'User Manager' do
+    before(:each) do
+      user = FactoryBot.create(:user_manager_user)
+      login_as(user, scope: :user)
+      stub_cognito_response(method: :list_users_in_group, payload: cognito_users)
+    end
+
+    it 'shows team members' do
+      visit update_user_path(user_id: member_user_id)
+      expect(page).to have_title t('users.show.title', name: member_first_name + ' ' + member_family_name)
+      expect(page).to have_content member_first_name + ' ' + member_family_name
+    end
+  end
+
+
+end
```

Comment:
~spec/system/visit_users_details_page.rb~ => spec/system/visit_users_details_page_spec.rb

Rspec might not have been running this test, I _think_ it looks for files ending in `_spec`

Reply:
It does, but I had troubles running it locally through the Code extension so I already prepared a follow-up branch to rename it.. since I didn't want to make this PR too confusing, but will update it now..

Reply:
fixed here https://github.com/alphagov/verify-self-service/pull/234/commits/953603d58f1930eed5bac35c31edcdb8211fdeeb along with one of your tests too 🙈